cliquet/utils.py

```diff
@@ -299,7 +300,13 @@ def follow_subrequest(request, subrequest, **kwargs):
               if no redirection happened.)
     """
     try:
-        return request.invoke_subrequest(subrequest, **kwargs), subrequest
+        try:
+            return request.invoke_subrequest(subrequest, **kwargs), subrequest
+        except Exception as e:
```

Comment:
It was an Exception here so I let it the same: https://github.com/mozilla-services/cliquet/pull/656/files#diff-00545b1c8fc056bc9e4acb9be5d68f42L108